### ISSUE TITLE
Issues with Wildrig on ubuntu with AMD 

### ISSUE THREAD

**Issue by realsetvin** [RainbowMiner/RainbowMiner#2380]
All wildrig algos report wattage incorrectly (as 0) on my AMD card, ranging from 580 to 6600. This affects profitability considerations and falsely encourages mining with wildrig 

I realize I can disable wildrig, however this will exclude a large amount of potential coins which could be mined. 

For example right now everprogpow seems popular - the wildrig miner reports 0 watts but also gets the memory junction of my 6600 beyond 94c.

 Using nano miner on everprogpow, I get accurate wattage reporting and also the memory junction is stable around 86c . I know this because nanominer is the fallback miner and wildrig often gets remove due to the below issue.

Wildrig also has frequent instances of "0 reported hashrate" which I suspect may be do to instances of dev fee cycles.

This leads me to my thought on what could be improved.

Can there be some fail safe written, that if reported wattage is 0 then your profit switching algorithm should just "guess" or take the reports from another miner to get within the range of accuracy of what the power cost should or would cost? 

Anyone else with AMD cards getting wildrig showing 0 wattage on the power section?

**Comment by RainbowMiner** [comment]
Yes, that is still a problem with the Wildrig miner. It just doesn't report Watts. Your idea to at least use some averages for the power consumption sounds pretty intriguing. I will give this a shot.

**Comment by realsetvin** [comment]
Had another idea which may require less math. Isnt there direct access to the gpu and cpu temps on board as well? All of my rigs have access to the apt "lm-sensors" by just installing it and then I can access "watch sensors" to check all my watts and temps. Could you pull that data?

Is it an issue with AMD cards? 

It might be more efficient to just directly call the temp of the 

Unsure if possible to do on windows, but seems like it could be easy on linux.

**Comment by RainbowMiner** [comment]
Duplicate of #2768, which tracks the AMD 0-watt power readings on Linux (and is referenced from this issue). The underlying bug is not fixed - it stays open in #2768.
